### ISSUE TITLE
Problem when exporting polyline from file geodatabase

### ISSUE THREAD

**Issue by boesiii** [tomchadwin/qgis2web#659]
I am trying to use data from a file geodatabase.  When I run the plugin I get an error:
```
"C:/Users/name/.qgis2/python/plugins\qgis2web\utils.py", line 127, in writeTmpLayer
uri = TYPE_MAP[layer.wkbType()]
KeyError: 2005
```
If I try exporting using points or polygons the export works as expected.

I am using Qgis 2.18.18

**Comment by boesiii** [comment]
I see that the error is saying that my geometry type is 2005 which is MultiLineStringM
https://qgis.org/api/qgswkbtypes_8h_source.html

Is the export not able to handle this type of geometry?

**Comment by boesiii** [comment]
I converted it to a MuliLineString and it is now working.

**Comment by tomchadwin** [comment]
Have a look in utils.py - you can see the supported geometry types, and add your own. I'll add this one to master.

**Comment by boesiii** [comment]
I see now.  Why not include all geometry types?  Is it because OL and LL don't support all those geomtery types?

**Comment by tomchadwin** [comment]
No, it's just because new ones were added with new versions of QGIS, and I've not kept up. Pull request welcome...

**Comment by boesiii** [comment]
Why are some TYPE_MAP `QGis.WKB...` and others are `QgsWKBTypes..`?

**Comment by tomchadwin** [comment]
I think it was because of an API change, but can't remember for sure.

Your fix is now merged, and released as 3.2.0. Many thanks!
